app/gosns/publish.go

```diff
@@ -86,51 +86,51 @@ func publishSQS(subscription *app.Subscription, topicName string, requestBody *m
 	arnSegments := strings.Split(queueName, ":")
 	queueName = arnSegments[len(arnSegments)-1]
 
-	if _, ok := app.SyncQueues.Queues[queueName]; ok {
-		msg := app.Message{}
+	if _, ok := app.SyncQueues.Queues[queueName]; !ok {
+		log.Infof("%s: Queue %s does not exist, message discarded\n", time.Now().Format("2006-01-02 15:04:05"), queueName)
```

Comment:
It's a good idea, but it's not our universal pattern (I know you probably noticed it below in the existing code) to include the time in there at the front.  So let's hold on that for now.  I will consider updating all the logs to fit that format in the meantime.